code/script.js

```diff
@@ -2,6 +2,10 @@
 const board = document.getElementById('board')
 const questions = document.getElementById('questions')
 const restartButton = document.getElementById('restart')
+const findOut= document.getElementById('filter')
+const winOrLose = document.getElementById('winOrLose')
```

Comment:
I think the win or lose section wasn't displaying because of this line - there is no winOrLose ID in the HTML, but there is a div with a class named win-or-lose. I think your code further down could have been solved by:

`const winOrLose = document.getQuerySelector('.win-or-lose')`